src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/InstallPackageCommand.cs

```diff
@@ -85,18 +85,18 @@ protected override void ProcessRecordCore()
             });
 
             // stop timer for telemetry event and create action telemetry event instance
-            TelemetryUtility.StopTimer();
-            var actionTelemetryEvent = TelemetryUtility.GetActionTelemetryEvent(
+            TelemetryServiceUtility.StopTimer();
+            var actionTelemetryEvent = VSTelemetryServiceUtility.GetActionTelemetryEvent(
```

Comment:
what was reasoning behind specific utility class for VS?

Reply:
we are using this service in current code, I just refactored this